src/components/BorrowerProfile/LendersAndTeams.vue

```diff
@@ -0,0 +1,233 @@
+<template>
+	<section>
+		<!-- Display Content  v-if="!loading && this.totalItemCount" -->
+		<div>
+			<h2 class="tw-text-h2">
+				{{ sectionTitle }}
+			</h2>
+
+			<div class="tw-mb-3">
+				<kv-material-icon
+					class="tw-h-2.5 tw-pointer-events-none tw-inline-block tw-align-middle"
+					:icon="mdiLightningBolt"
+				/>
+
+				<kv-loading-placeholder
+					v-if="loading"
+					:style="{width: 20 + (Math.random() * 15) + '%', height: '14px'}"
+					class="tw-ml-1 tw-inline-block tw-align-middle"
+				/>
+				<span v-else>
+					{{ poweredByText }}
+				</span>
+			</div>
+
+			<div v-if="loading" class="tw-grid tw-grid-cols-2 tw-gap-2">
+				<!-- Loading placeholder for team or lender item elements -->
+				<div v-for="i in 5" :key="i" class="tw-flex tw-flex-col md:tw-flex-row md:tw-items-center tw-mb-3">
+					<kv-loading-placeholder class="tw-rounded md:tw-hidden tw-mb-1.5" :style="{height: '128px'}" />
+					<kv-loading-placeholder
+						class="tw-rounded md:tw-block tw-hidden tw-mr-2" :style="{width: '88px', height: '88px'}"
+					/>
+					<div :style="{display: 'block', height: 'auto', width: '50%'}">
+						<kv-loading-placeholder class="tw-mb-1 tw-w-full" :style="{height: '14px'}" />
+						<kv-loading-placeholder :style="{width: 40 + (Math.random() * 15) + '%', height: '14px'}" />
+					</div>
+				</div>
+			</div>
+
+			<div v-if="!loading" class="section-items tw-mb-3 tw-grid tw-grid-cols-2 tw-gap-2">
+				<!-- Replace with Generic component for showing Either Team or Lender image and information -->
+				<div
+					v-for="(item, index) in truncatedItemList" :key="index"
+					class="tw-flex tw-flex-col md:tw-flex-row md:tw-items-center tw-mb-3"
+				>
+					<kv-loading-placeholder class="tw-rounded md:tw-hidden tw-mb-1.5" :style="{height: '128px'}" />
+					<kv-loading-placeholder
+						class="tw-rounded md:tw-block tw-hidden tw-mr-2" :style="{width: '88px', height: '88px'}"
+					/>
+					<div>
+						<span v-if="item.name">{{ item.name }}</span><br>
+						<span v-if="item.lenderPage && item.lenderPage.whereabouts">
+							{{ item.lenderPage.whereabouts }}
+						</span>
+					</div>
+				</div>
+			</div>
+
+			<div
```

Comment:
Please use a `button` tag to set the correct role and allow keyboard focus

Reply:
Will do in the follow up ticket for building out the lightbox.